packages/logical/logical.0.1.0/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+synopsis: "Logical is a minimalistic logic programming inspired by microKanren"
+description: """
+Logical is a minimalistic logic programming inspired by microKanren, which is
+- Simple implementation with only a few building blocks
+- Easy to understand and use
+- Supports negation free constraint logic programming
+"""
+maintainer: "Robert Toth <kkdstryker@gmail.com>"
+authors: ["Robert Toth"]
+homepage: "https://github.com/StrykerKKD/Logical"
+bug-reports: "https://github.com/StrykerKKD/Logical/issues"
+dev-repo: "git+https://github.com/StrykerKKD/Logical.git"
+doc: "https://strykerkkd.github.io/Logical/"
+license: "MIT"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "build" "@doc" "-p" name] {with-doc}
+]
+run-test: [
+  ["dune" "runtest" "-p" name "-j" jobs]
+]
+depends: [
+  "dune" {build}
+  "alcotest" {with-test}
+  "odoc" {with-doc}
+  "base" {>= "0.12.0"}
+  "ocaml" {>= "4.04.2" & < "4.08.0"}
```

Comment:
Are you sure it doesn't work on 4.08?

Reply:
Base is failing on 4.08.0 (and also Base has the same restriction), so I had to put it there.